_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -147,11 +147,11 @@ Even though a correct `<title>` element is put in the `<head>` of the document,
 
 #### Inapplicable Example 1
 
-This document has a `<title>` element but is inapplicable since the document element is an SVG `svg` element.
+This `title` element is a child of an `svg` element.
 
 ```svg
 <svg xmlns="http://www.w3.org/2000/svg">
-  <title>This is a circle</title>
+  <title>This is a red circle</title>
```

Comment:
the fill is actually green, does it matter?

Reply:
I debated whether or not to use a correct description or an incorrect description when tweaking the `title` element. The example is inapplicable, so I opted for an incorrect one to keep folks on their toes. It kind of doesn't matter.

Reply:
I think it's actually better to not change the code of the example inside this PR.
Changing the code of the example invalidate test cases AND requires a Final Call. Changing only the description is much lighter choice.

Additionally, we normally have Inapplicable examples that DO NOT fail the SC (reason: some tool can check the full SC without cutting it into smaller rules as we do, and the fail-on-fail relationship means that the "fail" result is the important one). It's not 100% clear to me right now whether a SVG with bad title fails WCAG or not, but in doubt I think we shouldn't do it in an Inapplicable example.